gobblin-utility/src/main/java/org/apache/gobblin/util/WriterUtils.java

```diff
@@ -284,13 +286,7 @@ public static void mkdirsWithRecursivePermissionWithRetry(final FileSystem fs, f
       return;
     }
 
-    if (path.getParent() != null && !fs.exists(path.getParent())) {
-      mkdirsWithRecursivePermissionWithRetry(fs, path.getParent(), perm, retrierConfig);
-    }
-
-    if (!fs.mkdirs(path, perm)) {
-      throw new IOException(String.format("Unable to mkdir %s with permission %s", path, perm));
-    }
+    gobblinMkDirs(fs, path, perm);
```

Comment:
Please update the javadoc in line 271

Reply:
The change itself doesn't change any behavior. What needs to be changed about the javadoc if the change is completely transparent?

Reply:
That's right actually. ignore my comment.